content/javascript/concepts/objects/terms/entries/entries.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.entries()'
+Description: 'Returns ..'
```

Comment:
I think this is unfinished or something?

Reply:
Sorry I forgot to update that, thanks for finding it! I added in a description.